src/traces/mesh3d/attributes.js

```diff
@@ -16,28 +16,57 @@ var extendFlat = require('../../lib/extend').extendFlat;
 module.exports = {
     x: {
         valType: 'data_array',
-        description: 'Sets the x coordinates of the vertices'
+        description: [
+            'Sets the X coordinates of the vertices. The nth element of vectors `x`, `y` and `z`',
+            'jointly represent the X, Y and Z coordinates of the nth vertex.'
+        ].join(' ')
     },
     y: {
         valType: 'data_array',
-        description: 'Sets the y coordinates of the vertices'
+        description: [
+            'Sets the Y coordinates of the vertices. The nth element of vectors `x`, `y` and `z`',
+            'jointly represent the X, Y and Z coordinates of the nth vertex.'
+        ].join(' ')
     },
     z: {
         valType: 'data_array',
-        description: 'Sets the z coordinates of the vertices'
+        description: [
+            'Sets the X coordinates of the vertices. The nth element of vectors `x`, `y` and `z`',
+            'jointly represent the X, Y and Z coordinates of the nth vertex.'
+        ].join(' ')
     },
 
     i: {
         valType: 'data_array',
-        description: 'Sets the indices of x coordinates of the vertices'
+        description: [
+            'A vector of vertex indices, i.e. integer values between 0 and the length of the vertex',
+            'vectors, representing the *first* vertex of a triangle. For example, `{i[m], j[m], k[m]}`',
+            'together represent face m (triangle m) in the mesh, where `i[m] = n` points to the triplet',
+            '`{x[n], y[n], z[v]}` in the vertex arrays. Therefore, each element in `i` represents a',
```

Comment:
This sentence ("each element in `i` represents a point in space") makes it sound like each element should be a triplet of xyz coordinates. Is that the case?